src/cli/ui/__tests__/components/common/Select.memo.test.tsx

```
@@ -16,47 +16,44 @@ import { Select, type SelectItem } from '../../../components/common/Select.js';
           <button
             data-testid="same-content"
             onClick={() => {
               // Create new array with same content
               setItems([
-                { label: 'Item 1', value: '1' },
-                { label: 'Item 2', value: '2' },
+                { label: "Item 1", value: "1" },
+                { label: "Item 2", value: "2" },
               ]);
             }}
           />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)